content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md

```diff
@@ -159,7 +159,29 @@ Node 1, Node 5, Node 2, Node 6, Node 3, Node 4
 
 After going over all the Nodes, it goes back to Node 1.
 
+## Enabling Opportunistic Batching
+
+{{< feature-state feature_gate_name="OpportunisticBatching" >}}
+
+When scheduling large workloads, pod definitions are typically identical and require scheduler
+to perform the same operations over and over again. The [Opportunistic Batching](/docs/reference/command-line-tools-reference/feature-gates/#OpportunisticBatching)
+feature allows scheduler to reuse the filtering and scoring results between scheduling cycles
+which greatly speeds up the scheduling process.
+
+The benefits come with certain limitations:
+1. Pods need to have equivalent definitions in terms of scheduling constraints
+1. Pods need to be scheduled at the same time (cache expires afer 0.5s) and need to directly follow one after another
+1. Only one batched pod can fit into one node
+1. Pods cannot use inter pod affinity/anti-affinity
+1. Pods cannot use topology spread constraints
+1. Pods cannot use DRA (cannot have any Resource Claims)
+1. The [DRAExtendedResource](/docs/reference/command-line-tools-reference/feature-gates/DRAExtendedResource.md) feature cannot be enabled.
+1. Scheduling profile needs to disable [default topology spread](/docs/concepts/scheduling-eviction/topology-spread-constraints/#internal-default-constraints)
+1. `IgnorePreferredTermsOfExistingPods` of [InterPodAffinityArgs](/docs/reference/config-api/kube-scheduler-config.v1/#kubescheduler-config-k8s-io-v1-InterPodAffinityArgs)
+can be set to `true` to make the batching more efficient.
```

Comment:
Let's explain those two separately? because they are "what a cluster admin has to configure their cluster", while others are "what kind of pods can get the benefits from this feature".

So, overall, I'd prefer the explanation to be like:
```md
This feature takes effect only for specific pods for now:
- No topology spread constraints.
- No DRA.
- ...

Also, to enable this feature in your scheduler, you have to configure the scheduler to:
- Change `IgnorePreferredTermsOfExistingPods` to `true`
- Set empty on default topology spread.

```